tasks.py

```diff
@@ -400,7 +400,7 @@ def bgp_dev_env(ip_family):
     try:
         # sudo because past docker runs will have changed ownership of this dir
         run('sudo rm -rf "%s"' % frr_volume_dir)
-        os.mkdir(frr_volume_dir)
+        os.makedirs(frr_volume_dir, exist_ok=True)
```

Comment:
Why we need this change?

Reply:
I've used this to create nested directory and this may not be needed if its set via parameter.

Reply:
this change is reverted now.